drivers/windows/inst/templates/task_run.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# automatically generated
+echo generated on host: {{hostname}}
+echo generated on date: {{date}}
+echo hipercow version: {{hipercow_version}}
+echo hipercow.windows version: {{hipercow_windows_version}}
+echo running on: $(hostname -f)
+
+export PATH=/opt/apps/lmod/lmod/libexec:$PATH
+source /opt/apps/lmod/lmod/init/bash
+export LMOD_CMD=/opt/apps/lmod/lmod/libexec/lmod
+module use /modules-share/modules/all
+
+module load R/{{r_version}}
+module load hiredis
+module load libsodium
+
+echo working directory: $(pwd)
```

Comment:
in the windows version the first bit of the script moves to the working directory. Is there some reason this won't work in linux? If not then we probably should move that bit of control into the same place on windows too